### PR TITLE
feat(Serial Podcast): add presence

### LINKED ISSUE #6808: Serial | serialpodcast.org
### Website name

Serial

### Website URL

https://SerialPodcast.org

### Website logo

https://media.wired.com/photos/59322cddaef9a462de98135a/master/w_1600%2Cc_limit/serial-logo-large.png

### Prerequisites

- [ ] It is a paid service
- [ ] It displays NSFW content
- [ ] It is region restricted

### Description

It would be nice if it displayed playback info such as: timestamp, season, episode, episode title

### PR DESCRIPTION
**Pull request by DiscordAnaxes**
## Description 

This pr implements and/or fixes the following:
- Resolves #6808 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary>Viewing an episode</summary>

image
</details>
<details>
<summary>Listening to an episode</summary>

image
</details>
<details>
<summary>Viewing a season</summary>

image
</details>
<details>
<summary>Reading a document</summary>

image
</details>
<details>
<summary>Reading a post</summary>

image
</details>

### CODE REVIEW

File: websites/S/Serial Podcast/presence.ts [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+				{
+					label: strings.buttonViewPage,
+					url: href,
+				},
+			],
+		},
+	};
+
+	for (const [path, data] of Object.entries(pages))
+		if (pathname.includes(path)) presenceData = { ...presenceData, ...data };
```
Comment (Bas950) [inline websites/S/Serial Podcast/presence.ts:111]: only 1 in there, just make it an if case

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/S/Serial Podcast/presence.ts (hunk 1 above)
Comment (DiscordAnaxes) [reply websites/S/Serial Podcast/presence.ts:111]: Will fix soon 👍

**Comment by EncryptedDev** [comment]
I've added the inactive tag until this is rectified

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Comment by EncryptedDev** [comment]
Please fix CI

**Pull request closed without merging**